examples/catboost/train.py

```diff
@@ -0,0 +1,35 @@
+# Based on the official regression example:
```

Comment:
@WeichenXu123 You can run this example, then see the logging result via `mlflow ui`.